src/main/asciidoc/shareddata.adoc

```diff
@@ -38,6 +38,9 @@ Here's an example of using a shared local map:
 Asynchronous shared maps allow data to be put in the map and retrieved locally when Vert.x is not clustered.
 When clustered, data can be put from any node and retrieved from the same node or any other node.
 
+In case that Vert.x is clustered, but a local asynchronous map is needed, you can call
```

Comment:
I would prefer to move this below the `IMPORTANT` callout and say that if you don't need the AsyncMap to be shared between nodes when Vert.x is not clustered, you can use `getLocalAsyncMap`.